src/opnsense/www/js/opnsense.js

```diff
@@ -254,7 +254,7 @@ function ajaxGet(url,sendData,callback) {
  */
 function watchScrollPosition() {
     function current_location() {
-        // concat url pieces to indentify this page and parameters
+        // concat url pieces to identify this page and parameters
```

Comment:
I presume that this file, unlike the preceding handful is actually native to this repository. -- As they're all in the same directory, a simple rule for skipping wasn't readily apparent.

I'm sorry about the noise. (I wrote this PR while babysitting a panicking linux server...)